### ISSUE TITLE
Some expansion possibilities

### ISSUE THREAD

**Issue by BoastfulCat** [if-ai/ComfyUI-IF_AI_tools#116]
I would like to ask if someone can give me some advice or if the author can make some small edits to the extension code:
- random value of presets prompt (embellish, style, negative)
- dynamic prompt. some like "{close-up, dramatic lighting, intense gaze into camera|full-body shot, dynamic pose, slightly tilted angle}"

regarding the dynamic prompt, I would like to implement the following logic (maybe this can be solved with some other extensions):
the main query generates an ollama. then, in each iteration, a line from the dynamic query is added to this query.

i.e. if the main prompt from the ollama is 

> Exquisite, ethereal beauty radiates from her porcelain skin, heart-shaped face, and captivating gaze, as she walks through an enchanted forest, adorned in a flowing, iridescent gown that shimmers like moonlight, her raven locks cascading down her back like a waterfall, surrounded by mythical creatures and lush flora, under a starry sky. Eerie, unsettling, dark, grotesque, suspenseful

and the dynamic prompt says 

> {close-up, dramatic lighting, intense gaze into camera|full-body shot, dynamic pose, slightly tilted angle}

then I should get several generations with queries like

queue 1:

> **close-up, dramatic lighting, intense gaze into camera**, Exquisite, ethereal beauty radiates from her porcelain skin, heart-shaped face, and captivating gaze, as she walks through an enchanted forest, adorned in a flowing, iridescent gown that shimmers like moonlight, her raven locks cascading down her back like a waterfall, surrounded by mythical creatures and lush flora, under a starry sky. Eerie, unsettling, dark, grotesque, suspenseful

queue 2:

> **full-body shot, dynamic pose, slightly tilted angle**, Exquisite, ethereal beauty radiates from her porcelain skin, heart-shaped face, and captivating gaze, as she walks through an enchanted forest, adorned in a flowing, iridescent gown that shimmers like moonlight, her raven locks cascading down her back like a waterfall, surrounded by mythical creatures and lush flora, under a starry sky. Eerie, unsettling, dark, grotesque, suspenseful

**Comment by if-ai** [comment]
I work into adding this.

**Comment by BoastfulCat** [comment]
I made a chain of different nodes so that i can use ollam's prompt and the variant combinator (https://github.com/adieyal/comfyui-dynamicprompts | https://github.com/florestefano1975/comfyui-prompt-composer). But! I can't figure out how to skip the repeated negation of ollam's prompt until all variants of the combinator are tried

**Comment by if-ai** [comment]
Yes I have added a new way where you get AI negatives in the coming update and then I will add the functionality you asked. so the negatives won't repeat as much

**Comment by if-ai** [comment]
![thorium_TkqmCtRVbb](https://github.com/user-attachments/assets/9c6bb771-dc78-4735-82c1-7983a71d8294)
There is a new wildcard node works similar to A1111 syntax you add your wildcard ymal files to the presetfolder and it will pick the category base on it's name 
here is the rules 
```
With WilcardMode enable
1-. Enter the text prompt using the categoy name __Category__ to indicate the field that should be replace by values from the wildcards.yaml or .json file
2. Use __Category/*__ to select rendom value from the item list use __Category/**__ to select the random value from within the subdirectories
3. Enter the dynamic_prompt select the variable name for the category then enter the equal simbol= and the values between curly brackets{} values separated by pipes| Color={red|blue|white|black} the script will create a prompt for each values overriding the values contained on the wildcards yaml or json file for the specific categories variables
4. select the maximun variant prompts you need with max_variants counter connect the text_list to a display text node
5. Execute the node
```
I also added a way to get __Prefix__ and __Suffix__  on the embelishment and styler son when you create a prompt you can automatically pass it trough the new wildcrad node to change those areas around the prompt
